aerogear-windows-otp/Digits.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace AeroGear.OTP
+{
+    public enum Digits
```

Comment:
Refactored `Digits` as an independent enumeration so one may use it generating other kinds of OTP.